build/configs/artik053/README.md

```diff
@@ -198,7 +198,42 @@ you must modify partition_map.cfg matching the sizes in ARTIK053_FLASH_PART_LIST
 
 will be updated
 
+#### audio
+ network, fs, pm, tash and audio are enabled.
+
+#### extra
+ network, fs, pm and tash are enabled.
+
+#### hello
+ hello example is enabled.
+
+#### iotivity
+ network, fs, romfs, pm, tash and iotivity are enabled.
+
+#### iotjs
+ network, fs, romfs, pm, logm, tash and iotjs are enabled.
+
+#### kernel_sample
+ fs, pm, tash and kernel_sample are enabled.
+
+#### minimal
+ fs and hello example are enabled.
+
 #### nettest
+ tash, fs, pm and network are enabled.
+
+#### st_things
+ network, fs, pm, iotivity, tash and st_things are enabled.
 
 #### tash
+ fs and tash are enabled.
+
+#### tc
+ network, fs, arastorage, pm, tash and testcase examples are enabled.
+
+#### typical
+ network, fs, pm and tash are enabled.
```

Comment:
1. Please keep same sequence.
for nettest
tash, fs, pm and network
for typical
network, fs, pm and tash

2. Is there no difference including app?